app/Http/Controllers/DeviceController.php

```diff
@@ -106,35 +106,46 @@ public function edit($id)
     public function update(EditDevice $request, $id)
     {
         $device = Device::findOrFail($id);
+        $site_id = null;
+        $location_id = null;
```

Comment:
Variable $location_id is not used before re-assignment.

Reply:
Fixed dddaf65